gui/src/main.cpp

```diff
@@ -84,57 +84,96 @@ int real_main(int argc, char *argv[])
 
 	QStringList cmds;
 	cmds.append("stream");
+	cmds.append("show");
 #ifdef CHIAKI_ENABLE_CLI
 	cmds.append(cli_commands.keys());
 #endif
 
 	parser.addPositionalArgument("command", cmds.join(", "));
-	parser.addPositionalArgument("host", "Address to connect to (when using the stream command)");
+	parser.addPositionalArgument("host", "Address to connect to (when using the stream command and don't want to use the settings (IP, morning, etc.) saved in the configuration file)");
 
 	QCommandLineOption regist_key_option("registkey", "", "registkey");
 	parser.addOption(regist_key_option);
 
 	QCommandLineOption morning_option("morning", "", "morning");
 	parser.addOption(morning_option);
 
+	QCommandLineOption fullscreen_option("fullscreen", "Start window in fullscreen (only for use with stream command)", "true");
```

Comment:
```suggestion
	QCommandLineOption fullscreen_option("fullscreen", "Start window in fullscreen (only for use with stream command)");
```
If you remove the value, you will get a simple boolean toggle which you can pass like `--fullscreen` (without =true).
You can then later check it like `parser.isSet(fullscreen_option)`.

Reply:
Thanks, I actually searched for exact this solution, but was missing the 'isSet' part. I will apply this.